CoEpi/api/Api.swift

```diff
@@ -50,18 +50,16 @@ class ApiImpl: Api {
     }
 
     func getCenKeys() -> Single<[String]> {
-        let unixDT = Int(currentAttemptedExposureCheck.timeIntervalSince1970)
-        let stringURL = "https://coepi.wolk.com:8080/cenkeys/\(unixDT)"
+        let stringURL = "https://coepi.wolk.com:8080/cenkeys/\(Int(currentAttemptedExposureCheck.timeIntervalSince1970))"
         return json(.get, stringURL).asSingle().map { result in
             result as! [String]
         }
     }
 
-    func getCenReport(cenKey: CENKey) -> Single<CENReport> {
+    func getCenReport(cenKey: CENKey, id: String, report: String) -> Single<CENReport> {
         let stringURL = "https://coepi.wolk.com:8080/cenreport/\(cenKey)"
         return json(.get, stringURL).asSingle().map { result in
-            // TODO actual result
-            CENReport(id: "1", report: "report1", timestamp: Int64(Date().timeIntervalSince1970))
+            CENReport(id: id, report: report, timestamp: Int64(Date().timeIntervalSince1970))
```

Comment:
This request is to retrieve the report, using the key. This is not what it's doing.